### ISSUE TITLE
How to use icons prop from v.1.4.0?

### ISSUE THREAD

**Issue by nikolailehbrink** [emilkowalski/sonner#319]
Hi, I came across #270 a couple of days ago and was hyped that the change is integrated. Now today I saw the release of 1.4.0 and wanted to use the icons prop, as I wanted to set default icons for states like "success" and "error" etc.

However after trying out like mentioned in the pull request, I couldn't get it to work and the types were also not detected.
```js
/**
   * @deprecated Please use the `icons` prop instead:
   * ```jsx
   * <Toaster
   *   icons={{ loading: <LoadingIcon /> }}
   * />
   * ```
   */
```
After looking further in #294 I saw that you removed the types and did some changes, so my question is pretty simple. How can I use the icons prop that was merged with #270 ?

Kind regards and thanks for the work!

**Comment by amirhhashemi** [comment]
> How can I use the icons prop that was merged with https://github.com/emilkowalski/sonner/pull/270 ?

You can't, yet.

@emilkowalski Can I ask why you reverted #270? I even locally tested the package didn't encounter any issues.

**Comment by emilkowalski** [comment]
@amirhhashemi There was a type error that caused all toasts with types to crash, if Toaster didn't have the icons prop. You can check your PR and click on any toasts with a type to see it happening if you are curious. I've fixed that and brought it back in #325.
